cookie-policy.md

```diff
@@ -4,8 +4,6 @@ slug: /information/tos/cookie-policy/
 ---
 # Cookie Policy
 
-Last updated: May 25th, 2018
```

Comment:
shouldn't we keep (update) this one? We do have it for the other policy documents

Reply:
I'm being remote controlled, can you bring up concerns here: https://wherebyhq.slack.com/archives/CMK9L1W05/p1597236394306800

Don't think Rory has Github access.